celery/worker/consumer/tasks.py

```diff
@@ -91,12 +90,17 @@ def qos_global(self, c) -> bool:
 
         if c.app.conf.worker_detect_quorum_queues:
             using_quorum_queues, qname = self.detect_quorum_queues(c)
+
+            if c.app.conf.broker_native_delayed_delivery is True:
```

Comment:
Does this mean the new flag implicitly enables quorum queue mode a.k.a global equal false?

Because the purpose of the original code is to validate against the queues args.

Won't it be confusing?

Thinking out loud

Reply:
You're right. That's no longer the case.